internal/common/conf.go

```diff
@@ -14,9 +14,13 @@ package common
 import "github.com/spf13/viper"
 
 const (
-	configType              string = "yaml"      // we are only capable of using yaml config
-	primaryConfigLocation   string = "./config/" // we are expecting our yaml file to be in a config folder right aside the binary
-	secondaryConfigLocation string = "."         // if not, the config file might directly be aside the binary
+	DbDriverName              string = "database.driverName"
+	DbUrl                     string = "database.url"
+	defaultDatabaseDriverName string = "sqlite3"
```

Comment:
why not having it directly in the config file? this way the user can change the database name

Reply:
I thougth it is an elegant way of configuring it. I can change if you dislike it.

Reply:
The issue is, that the user will not have any kind of option if the file needs to be at a different location for some reason.

Reply:
Have changed it